src/services/domainService.js

```diff
@@ -70,7 +68,13 @@ async function createSSLDirectory() {
 async function generateAndReplaceMainApplicationHaproxyConfig() {
   try {
     // get applications on the network
-    const applicationSpecifications = await fluxService.getAppSpecifications();
+    let applicationSpecifications = await fluxService.getAppSpecifications();
+
+    // If there's ownersApps, only include them
+    if(config.ownersApps.length > 0) {
```

Comment:
we can do just 'config.ownersApps.length'
+ space for lint

Reply:
Done! Thanks!